install.sh

```diff
@@ -66,23 +83,49 @@ https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=0&mimet
 EOF
                 echo "[i] File created."
 
-                # add cronjob
-                read -r -p "[?] Create a cronjob which updates /etc/hosts with new adservers every 5 days? [Y/n] " REPLY
+                # add systemd service
+                read -r -p "[?] Create a systemd service which updates /etc/hosts with new adservers every week? [Y/n] " REPLY
                 case "${REPLY}" in
                     "YES" | "Yes" | "yes" | "Y" | "y" | "" )
-                        echo "[i] Creating cronjob..."
```

Comment:
I think it was nice to install either a systemd timer or a cronjob. There are still some distros out there without systemd (even if I don't understand why)...